build/codegen/codegen.sh

```diff
@@ -14,29 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+set -o pipefail
+set -o nounset
+set -o errexit
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Comment:
Script error setup is not related to determining the script's dir. Separate these lines

```suggestion
set -o errexit

script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```